setup.py

```diff
@@ -111,10 +98,8 @@ def solve_transitive_dependencies(cython_extensions):
                 ]
                 if not_cython_files:
                     raise Exception(
-                        "Extension %s was finished but a dependency isn't a cython file: %s" % (
-                            extension["name"],
-                            not_cython_files
-                        )
+                        "Extension %s was finished but a dependency isn't a cython file: %s"
+                        % (extension["name"], not_cython_files)
```

Comment:
This should be like the original code (it's the style I follow for multiline formats)